src/seedu/addressbook/AddressBook.java

```diff
@@ -31,6 +32,10 @@
  **/
 public class AddressBook {
 
+    private static final String FILE_EXTENSION_DELIMETER = ".";
+
+    private static final int EMPTY_FILE_NAME = 0;
```

Comment:
This doesn't help either. If used in only one place, better to explain it locally where it is used.